js-pkg/client/src/provider.ts

```diff
@@ -407,6 +422,17 @@ export class YSweetProvider {
     }
 
     this.isConnecting = false
+
+    if (this.showDebuggerLink && lastDebugUrl !== this.debugUrl) {
+      console.log(
+        `%cOpen this in Y-Sweet Debugger ⮕ ${this.debugUrl}`,
+        'font-size: 1.5em; display: block; padding: 10px;',
+      )
+      console.log(
+        '%cTo hide the debugger link, pass showDebuggerLink={false} to YDocProvider',
```

Comment:
Now that this is in the client we can't assume the user is using react, right? Maybe we need to cover both cases (react and native) here?

Reply:
good catch, I'll just make the message a bit more generic